src/tools/spin/spinquic.cpp

```diff
@@ -215,6 +215,33 @@ QUIC_STATUS QUIC_API SpinQuicHandleStreamEvent(HQUIC Stream, void * /* Context *
 QUIC_STATUS QUIC_API SpinQuicHandleConnectionEvent(HQUIC Connection, void * /* Context */, QUIC_CONNECTION_EVENT *Event)
 {
     switch (Event->Type) {
+    case QUIC_CONNECTION_EVENT_CONNECTED: {
+        int Selector = rand() % 3;
```

Comment:
Instead of calling `rand()` directly, can you use the helper from above (that everything else uses)?